src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -158,14 +159,26 @@ async Task OnMessage(string method, JObject args, CancellationToken token)
                 case "Runtime.consoleAPICalled":
                     _logger.LogInformation(FormatConsoleAPICalled(args));
                     break;
+                case "Inspector.targetCrashed":
+                case "Runtime.exceptionThrown":
+                    _logger.LogWarning($"Failing all waiters because: {method}: {args}");
+                    fail = true;
+                    break;
+                case "Debugger.scriptParsed":
+                case "Debugger.resumed":
+                case "Debugger.breakpointResolved":
+                case "Runtime.executionContextCreated":
+                    break;
+                default:
+                    _logger.LogWarning(method);
+                    break;
```

Comment:
Why are these needed?

Reply:
I wanted to see any unknow in the log as warning. So, those which are known to be OK, should not spam the log.

Reply:
This doesn't expect to handle every message, so this doesn't need to be a warning. For example, you could have messages that are being waited upon via `On(..)`, which gets handled at line 178, and those should not generate a warning.

Reply:
improved